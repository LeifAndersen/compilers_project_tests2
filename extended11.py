x = [i for i in range(10)]
y = [i*2 for i in range(100) if i < 20 or i > 40]
z = {z for z in range(10)}
x = {z:2 for z in range(50)}

