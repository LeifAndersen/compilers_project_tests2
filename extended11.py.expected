(program
 (= (x) (list i (for i (indexed range (called 10)) ())))
 (=
  (y)
  (list
   (term i ("*" 2))
   (for
    i
    (indexed range (called 100))
    (if (or (comparison i ("<" 20)) (comparison i (">" 40))) none))))
 (= (z) (set z (for z (indexed range (called 10)) ())))
 (= (x) (dict (z 2) (for z (indexed range (called 50)) ()))))

